### PR TITLE
working on generate a new travel

### PR DESCRIPTION
**Pull request by fruizrob**

### CODE REVIEW

File: app/Controller/ActionController.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+namespace Controller
+{
+    public class ActionController
+    {
+        public static void FeedActionsComboBox(ComboBox combo_box)
+        {
+            combo_box.DisplayMember = "description";
+            combo_box.ValueMember = "action_id";
+            combo_box.DataSource = Action.FindAll();
```
Comment (geraldbustosm) [inline app/Controller/ActionController.cs:13]: All()

File: app/Controller/TravelController.cs [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+        public int GetLastTravelSection()
+        {
+            // lo necesito para almacenar las acciones de las secciones antes de ser ingresadas a la base de datos
+            TravelSection last_travel_section = TravelSection.GetLastTravelSection();
+            int id = (last_travel_section != null) ? last_travel_section.travel_id : 1;
+            return id;
+        }
+
+        public bool AddNewActionToSection(string action_description, string patent, string type)
```
Comment (geraldbustosm) [inline app/Controller/TravelController.cs:85]: AddNewActionToMachineInSection() algo asi

File: app/Model/Action.cs [hunk 3]
```diff
@@ -1,102 +1,98 @@
...
+                            int minutes = reader.GetInt32(2);
+                            action = new Action(description, minutes);
+                            action.action_id = id;
+                        }
+                    }
+                }
             }
-            connection.Close();
-            return null;
+            return action ?? null;
```
Comment (fruizrob) [inline app/Model/Action.cs:69]: return action;

File: app/Model/Action.cs [hunk 4]
```diff
@@ -1,102 +1,98 @@
...
+                            Action action = new Action(description, minutes);
+                            action.action_id = id;
+                            all_actions.Add(action);
+                        }
+                    }
+                }
             }
-            connection.Close();
-            if (count > 0) { return true; } else { return false; }
+            return all_actions ?? null;
```
Comment (fruizrob) [inline app/Model/Action.cs:95]: return all_actions;

**Pull request merged**